docs/flink_job_cluster_guide.md

```diff
@@ -0,0 +1,133 @@
+# Create Flink job clusters with Helm Chart
+
+## Overview
+
+This Helm Chart is an addition to the [existing](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/tree/master/config/samples) way of deploying Flink job clusters.
+
+### Why use the Helm Chart?
+- Using this Chart enables you to easily configure many Flink job clusters based on a single template
+- It enables you to use configuration manifests that belong to each other from a central place e.g. ```PodMonitor```
```

Comment:
I don't quite understand what this means, could you rephrase?

Reply:
I added a few more lines, to be honest in the current state of the Flink job cluster YAML, you could very well deploy it without the helm chart. However, as more manifest files are added that are deployed togheter with the Flink job cluster, such as the PodMonitor, it'll become easier to just enable them or configure them in one values.yaml. The biggest pro for us are the pre deployment checks that are included in helm as well as the post deployment check and automated rollback with --atomic.